index.html

```diff
@@ -0,0 +1,173 @@
+            <div class="event-line">
+                <h2 class="event-greeting">"Hello! Sharing world"</h2>
+                <h1 class="rainbow-text">PROUD 2 PAY MARKETING CAMPAIGN EVENT 2023</h1>
+                <p>“We have technology, finally, that for the first time in human history <span id="content"></span> allows people to really maintain rich connections with much larger numbers of people.”</p>
+            </div>
+            <div class="event-date">
+                <h2>2023.08.23(WED) ~ 25(FRI)</h2>
+                <p>@Zoom Meet</p>
+            </div>
+        </section>
+
+        <!-- Feature -->
+        <section class="program" id="program">
+            <h2 class="program-text">Main Program</h2>
+            <span class="highlight"></span>
+            <div class="program-list">
+                <!-- Card 1 -->
+                <div class="program-card">
+                    <img src="imgs/distribution-icon.png" alt="Marketplace">
+                    <h3 class="program-title">Distribution</h3>
+                    <p>“The best supply chain is one that has no beginning and no end.”</p>
+                </div>
+                <!-- Card 2 -->
+                <div class="program-card">
+                    <img src="imgs/digital-marketing.png" alt="Affiliate">
+                    <h3 class="program-title">Marketing</h3>
+                    <p>“Business has only two functions — marketing and innovation.”</p>
+                </div>
+                <!-- Card 3 -->
+                <div class="program-card">
+                    <img src="imgs/promotion.png" alt="blog">
+                    <h3 class="program-title">Promotion</h3>
+                    <p>“It's not what you sell that matters as much as how you sell it!”</p>
+                </div>
+                <!-- Card 4 -->
+                <div class="program-card">
+                    <img src="imgs/educate.png" alt="Education">
+                    <h3 class="program-title">Education</h3>
+                    <p>“Education is one thing no one can take away from you.”</p>
+                </div>
+            </div>
+            <a href="#" class="program-text" type="button"> SEE THE WHOLE PROGRAM</a>
+        </section>
+
+        <!-- <section class="blog">
+            <div class="rightimg">
+                <div class="blog1">
+                    <img src="imgs/video-conference-meeting.jpg" alt="">
+                </div>
+                <div class="blog1">
+                    <img src="imgs/crowd-participating-at-event.jpg" alt="">
+                </div>
+            </div>
+            <div class="leftimg">
+                <div class="blog1">
+                    <img src="imgs/video-conference-meeting.jpg" alt="">
+                </div>
+                <div class="blog1">
+                    <img src="imgs/crowd-participating-at-event.jpg" alt="">
+                </div>
+            </div>
+        </section> -->
```

Comment:
Please kindly remove unused code from the codebase. it is not good to commit old piece of code as inline comments. They will make your project look messy. if you need to review a previous version of your code, you can always use git history.

Reply:
Hello, I've deleted the codes commented out.